src/panels/lovelace/cards/hui-vertical-stack-card.ts

```diff
@@ -0,0 +1,102 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import computeCardSize from "../common/compute-card-size.js";
+import createCardElement from "../common/create-card-element.js";
+
+import { LovelaceCard, LovelaceConfig } from "../types.ts";
+
+interface Config extends LovelaceConfig {
+  cards: LovelaceConfig[];
+}
+
+class HuiVerticalStackCard extends LitElement
+  implements LovelaceCard {
+  protected hass?: HomeAssistant;
+  protected config: Config;
+  private elements: LovelaceCard[];
+
+  static get properties() {
+    return {
+      hass: {
+        observer: "_hassChanged",
```

Comment:
This doesn't exist in lit.

Reply:
You need to remove `hass` as a property that causes lit to re-render. Make your own updater

```
set hass(value) {
    this._hass = hass;
    this.shadowRoot.querySelectorAll('#root > *').forEach(el => { el.hass = hass; });
}
```

Reply:
```
set hass(hass: HomeAssistant) {
    this.hass = hass;
    for (const el of this.shadowRoot.querySelectorAll('#root > *')) {
      el.hass = hass;
    };
  }
```

This appears to put me an infinite loop where updating `hass` of the element updates `hass` in general

Reply:
this.hass = hass; is your infinite loop

Reply:
You will need to remove it as a property

Reply:
oh boy...I need more coffee for this job